### PR TITLE
simplified solution in module 12, added questions in module 11 and 12

### PR DESCRIPTION
**Pull request by CafeSamosaBlue**

### CODE REVIEW

File: problemsets-mat223/modules/module11-exercises.tex [hunk 1]
```diff
@@ -228,5 +228,22 @@
 					A)=2$ $\Nullity(T) = \Nullity(A)=1$
 			\end{enumerate}
 		\end{solution}
+
+		\prob Give an example of a $3x4$ matrix $M$ with the specified rank, or explain why it cannot exist.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module11-exercises.tex:232]: `\times` not `x`

it -> one

File: problemsets-mat223/modules/module11-exercises.tex [hunk 2]
```diff
@@ -228,5 +228,22 @@
...
+		\begin{enumerate}
+			\item $\Rank(M) = 0$
+			\item $\Rank(M) = 1$
+			\item $\Rank(M) = 3$
+			\item $\Rank(M) = 4$
+		\end{enumerate}
+
+		\prob Consider the linear transformation $P:\R^3\to\R^3$, the projection onto the $xy$-plane.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module11-exercises.tex:240]: This isn't grammatical.

File: problemsets-mat223/modules/module12-exercises.tex [hunk 3]
```diff
@@ -175,12 +175,14 @@
 			\item For an arbitrary linear transformation $T:\R^{m}\to\R^{n}$,
 				if $n \neq m$, then the linear transformation is
 				not invertible.
-
 			\item The matrix $M = \mat{1&0&0\\1&1&0}$ is an elementary
 				matrix.
 			\item Every elementary matrix is invertible.
 			\item The product of elementary matrices is sometimes an elementary matrix.
 			\item The product of elementary matrices is always an elementary matrix.
+			\item A matrix that induces an invertible linear transformation is not necessarily invertible.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module12-exercises.tex:183]: Remove the `not`

File: problemsets-mat223/modules/module12-exercises.tex [hunk 4]
```diff
@@ -175,12 +175,14 @@
...
+			\item A transformation that is one-to-one and onto is always invertible
+			\item For two Matricies $A$ and $B$, if $AB=I$ then $A$ and $B$ are invertible.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module12-exercises.tex:185]: lowercase m

**Comment by siefkenj** [comment]
Great! Can you also include solutions?

**Comment by siefkenj** [comment]
Travis says you have a missing $...

File: problemsets-mat223/modules/module11-exercises.tex [hunk 5]
```diff
@@ -228,5 +228,42 @@
...
+			\item $\Rank(M) = 3$
+			\item $\Rank(M) = 4$
+		\end{enumerate}
+
+		\begin{solution}
+			\begin{enumerate}
+				\item $\mat{0&0&0&0\\0&0&0&0\\0&0&0&0}$
+				\item $\mat{1&0&0&0\\0&0&0&0\\0&0&0&0}$
+				\item $\mat{1&0&0&0\\0&1&0&0\\0&0&1&0}$
+				\item A $3\times4$ matrix with rank 4 cannot exist. Two examples of justifications are: \\
```
Comment (siefkenj) [inline problemsets-mat223/modules/module11-exercises.tex:245]: Don't force newlines.

File: problemsets-mat223/modules/module11-exercises.tex [hunk 6]
```diff
@@ -228,5 +228,42 @@
...
+			\item Find the column space of $M_P$. Are there any similarities to your answer in the previous part?
+			\item Find the null space of $P$ and $M_P$. Are there similarities between the null space of a linear transformation and its associated matrix?
+		\end{enumerate}
+
+		\begin{solution}
+			\begin{enumerate}
+				\item $\mat{1&0&0\\0&1&0\\0&0&0}$
+				\item the range of $P$ is the $xy$-plane.
+				\item the column space of $M_P$ is $\Span\Set*{\mat{1\\0\\0},\mat{0\\1\\0}} =$ the $xy$-plane. The column space of a matrix is the same as the range of its induced transformation.
+				\item the null space of $P$ and $M_P$ is $\Span\Set*{\mat{0\\0\\1}} =$ the $z$-axis. The null space of a matrix is the same as the null space of its induced transformation.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module11-exercises.tex:264]: Sentences start with capital letters. Also, don't use `= <a word>`. Write it out in English in that case.

File: problemsets-mat223/modules/module12-exercises.tex [hunk 7]
```diff
@@ -175,12 +175,27 @@
...
+			\item A matrix that induces an invertible linear transformation is necessarily invertible.
+			\item A transformation that is one-to-one and onto is always invertible
+			\item For two matricies $A$ and $B$, if $AB=I$ then $A$ and $B$ are invertible.
 		\end{enumerate}
+
+		\begin{solution}
+			\begin{enumerate}
+				\item True. If $m<n$ then $T$ cannot be onto. If $m>n$ then $T$ cannot be one-to-one. In both cases, $T$ cannot be invertible. We may conclude that the dimension of the domain and codomain must be equal in order for a transformation to be invertible.
+				\item False. Elementary matricies are one row operation away from the identity matrix. The identity matrix of any size is always square. There is no identity matrix such that performing one row operation on it yields $M$. We may conclude that elementary matricies are always square.
+				\item True. The inverse of an elementary matrix $E$ is another elementary matrix $E^{-1}$ which corresponds to the row operation that turns $E$ into the identity matrix. $E^{-1}$ is the "opposite" row operation.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module12-exercises.tex:192]: Use ` for open quotes

File: problemsets-mat223/modules/module12-exercises.tex [hunk 8]
```diff
@@ -175,12 +175,27 @@
...
+				\item True. Let $E_1$ correspond to multiplying row 1 by 4. Let $E_2$ correspond to multiplying row 1 by 2. The product $E_2E_1$ corresponds to multiplying row 1 by 8, which is also a single row operation and thus has a corresponding elementary matrix.
+				\item False. Let $E_1$ correspond to multiplying row 1 by 4. Let $E_2$ correspond to multiplying row 2 by 2. The product $E_2E_1$ corresponds to multiplying row 1 by 4 and row 2 by 2, which is not a single row operation and thus does not have a corresponding elementary matrix.
+				\item True.
+				\item True.
+				\item False. Let $A = \mat{1&0&0\\0&1&0}$. Let $B = \mat{1&0\\0&1\\0&0}. $AB = I$. $BA = \mat{1&0&0\\0&1&0\\0&0&0} \neq I$. $A$ and $B$ are not invertible. It is required that $AB = BA = I$ in order for matrices $A$ and $B$ to be invertible.
+			\end{enumerate}
```
Comment (siefkenj) [inline problemsets-mat223/modules/module12-exercises.tex:198]: Run all this code through the latex beautifier so it linewraps.  https://siefkenj.github.io/latex-parser/

File: problemsets-mat223/modules/module11-exercises.tex [hunk 9]
```diff
@@ -228,5 +228,42 @@
...
+			\item $\Rank(M) = 4$
+		\end{enumerate}
+
+		\begin{solution}
+			\begin{enumerate}
+				\item $\mat{0&0&0&0\\0&0&0&0\\0&0&0&0}$
+				\item $\mat{1&0&0&0\\0&0&0&0\\0&0&0&0}$
+				\item $\mat{1&0&0&0\\0&1&0&0\\0&0&1&0}$
+				\item A $3\times4$ matrix with rank 4 cannot exist. Two examples of justifications are:
+				1) Thinking about the induced transformation, the 3 dimensional domain cannot produce a 4 dimensional range (Rank-Nullity Theorem).
```
Comment (siefkenj) [inline problemsets-mat223/modules/module11-exercises.tex:246]: Be more precise with your language here. Maybe spell out the argument. Every vector in the domain is a LC of e_1, e_2, e_3, so every vector in the range must be a LC of Ae_1, .... Thus the range cannot be more than three dimensional.

File: problemsets-mat223/modules/module11-exercises.tex [hunk 10]
```diff
@@ -228,5 +228,42 @@
...
+				2) A matrix with rank 4 has 4 pivots. A $3\times4$ matrix only has 3 rows which makes it impossible to have 4 pivots.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module11-exercises.tex:247]: rephrase to "can have at most 3 pivots, which is less than 4"

File: problemsets-mat223/modules/module11-exercises.tex [hunk 11]
```diff
@@ -228,5 +228,42 @@
...
+			\end{enumerate}
+		\end{solution}
+
+		\prob Let $P:\R^3\to\R^3$ be the projection onto the $xy$-plane.
+		\begin{enumerate}
+			\item Find a matrix $M_P$ for the transformation.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module11-exercises.tex:253]: the transformation -> `\mathcal P`

File: problemsets-mat223/modules/module11-exercises.tex [hunk 12]
```diff
@@ -228,5 +228,42 @@
...
+				\item $\mat{0&0&0&0\\0&0&0&0\\0&0&0&0}$
+				\item $\mat{1&0&0&0\\0&0&0&0\\0&0&0&0}$
+				\item $\mat{1&0&0&0\\0&1&0&0\\0&0&1&0}$
+				\item A $3\times4$ matrix with rank 4 cannot exist. Two examples of justifications are:
+				1) Thinking about the induced transformation, the 3 dimensional domain cannot produce a 4 dimensional range (Rank-Nullity Theorem).
+				2) A matrix with rank 4 has 4 pivots. A $3\times4$ matrix only has 3 rows which makes it impossible to have 4 pivots.
+			\end{enumerate}
+		\end{solution}
+
+		\prob Let $P:\R^3\to\R^3$ be the projection onto the $xy$-plane.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module11-exercises.tex:251]: Make it `\mathcal P` throughout to distinguish it from matrices.

File: problemsets-mat223/modules/module12-exercises.tex [hunk 13]
```diff
@@ -175,12 +175,52 @@
...
+
+				\item False. Let $E_{1}$ correspond to multiplying row 1 by 4. Let $E
+					_{2}$ correspond to multiplying row 2 by 2. The product $E_{2}E_{1}$
+					corresponds to multiplying row 1 by 4 and row 2 by 2, which is not a
+					single row operation and thus does not have a corresponding
+					elementary matrix.
+
+				\item True.
+
+				\item True.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module12-exercises.tex:217]: Are you going to provide arguments for these parts?

Comment (CafeSamosaBlue) [reply problemsets-mat223/modules/module12-exercises.tex:217]: No i am not

File: problemsets-mat223/modules/module11-exercises.tex [hunk 14]
```diff
@@ -228,5 +228,48 @@
...
+
+		\begin{solution}
+			\begin{enumerate}
+				\item $\mat{0&0&0&0\\0&0&0&0\\0&0&0&0}$
+
+				\item $\mat{1&0&0&0\\0&0&0&0\\0&0&0&0}$
+
+				\item $\mat{1&0&0&0\\0&1&0&0\\0&0&1&0}$
+
+				\item A $3\times4$ matrix with rank 4 cannot exist. Two examples of justifications
```
Comment (siefkenj) [inline problemsets-mat223/modules/module11-exercises.tex:248]: -`examples of`

File: problemsets-mat223/modules/module11-exercises.tex [hunk 15]
```diff
@@ -228,5 +228,48 @@
...
+			\item Find a matrix $M_\mathcal{P}$ for the transformation.
+			\item Find the range of $\mathcal{P}$.
+			\item Find the column space of $M_P$. Are there any similarities to your answer in the previous part?
+			\item Find the null space of $\mathcal{P}$ and $M_\mathcal{P}$. Are there similarities between the null space of a linear transformation and its associated matrix?
+		\end{enumerate}
+
+		\begin{solution}
+			\begin{enumerate}
+				\item $\mat{1&0&0\\0&1&0\\0&0&0}$
+				\item The range of $P$ is the $xy$-plane.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module11-exercises.tex:268]: `\mathcal`. You're missing it in a few other places too.

File: problemsets-mat223/modules/module12-exercises.tex (hunk 13 above)
Comment (siefkenj) [reply problemsets-mat223/modules/module12-exercises.tex:217]: Please do.

**Pull request merged**